docs/getting_started/using_modin/using_modin_cluster/using_modin_ray_cluster.rst

```diff
@@ -0,0 +1,125 @@
+================================
+Using Modin in a AWS Ray Cluster
+================================
+
+.. note::
+  | *Estimated Reading Time: 15 minutes*
+
+Often in practice we have a need to exceed the capabilities of a single machine.
+Modin works and performs well in both local mode and in a cluster environment.
+The key advantage of Modin is that your notebook does not change between
+local development and cluster execution. Users are not required to think about
+how many workers exist or how to distribute and partition their data;
+Modin handles all of this seamlessly and transparently.
+
+.. image:: ../../../img/modin_cluster.png
```

Comment:
This picture is not that good looking in the docs. Can we format it for better appearance?